data/planning/index.html

```diff
@@ -10,7 +10,13 @@
 title: Planning Overview
 categories: []
 ---
-  <table>
+{% include table_overview.html name="Planning" %} 
+
+<p>The list shown below includes dataset names, descriptions, and links to some of the most important statewide and regional data resources for transportation planning, land use planning, and economic development. It also includes some datasets that represent the geography of notable formal plans and proposals.</p>
+<p>The list is meant as a starting point and users are encouraged to reach out to agencies, local government, and further search across UGRC's State Geographic Information Database for additional existing data. The overall objective of keeping this list is to connect others to valuable resources efficiently and avoid duplicative work.</p>
+<p>If you have ideas for additions to this list, contact <a href="mailto:ugrc@utah.gov">ugrc@utah.gov</a> or <a href="mailto:analytics@wfrc.org">analytics@wfrc.org</a>.</p>
```

Comment:
**chore**: we don't accept mailto links directly in pages since they are a maintenance issue. We use this [contacts](https://github.com/agrc/gis.utah.gov/blob/master/_data/contacts.yml) dataset and an include. You can read about how to use it in the [readme](https://github.com/agrc/gis.utah.gov#contact-information).

If you'd like help you know where to find me.